documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -34,7 +34,14 @@ Consider a shorter interval for fast changing clusters, although this increases
 [discrete]
 == Contents of optimization proposals
 
-The following table describes the properties contained in an optimization proposal:
+An optimization proposal consists of two main sections: the _summary_ and the _broker load_.
+The summary provides an overview of the proposed cluster rebalance and indicates the scale of the changes involved.
+The broker load allows you to see the impact of the proposed rebalance on each of the brokers in the cluster.
```

Comment:
_Suggested change:_

The broker load allows you to see the impact of the proposed rebalance on each of the brokers in the cluster.

>

The broker load shows before and after values for the proposed rebalance, so you can see the impact on each of the brokers in the cluster.

Reply:
Okay. I will apply this change